pkgs/tools/security/open-fprintd/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, fetchFromGitHub, python38Packages }:
```

Comment:
Does this specifically need Python 3.8? If so, please add a comment.

Reply:
Not AFAIK, I will test. Should I prefer `pythonPackages` instead then?

Reply:
`python3.pkgs` is the preferred variant.

Reply:
Looks very similar to what I had: https://github.com/kurnevsky/nixfiles/blob/master/modules/open-fprintd.nix
It didn't build with the latest python AFAIR :)

Reply:
Or it might be https://github.com/kurnevsky/nixfiles/blob/master/modules/python-validity.nix that didn't build...